ArduSub/ArduSub.cpp

```diff
@@ -36,6 +36,7 @@ const AP_Scheduler::Task Sub::scheduler_tasks[] = {
     SCHED_TASK(three_hz_loop,          3,     75),
     SCHED_TASK(update_turn_counter,   10,     50),
     SCHED_TASK(compass_accumulate,   100,    100),
+    SCHED_TASK(update_beacon,         50,     50),
```

Comment:
The beacon sampling of the copter is 400 Hz. I think that this is from 2.5 ms of one cycle of the copter. Submarine is also 2.5 ms, so I think it will be 400 Hz.